hs_core/tests/api/native/test_create_new_version_resource.py

```diff
@@ -0,0 +1,198 @@
+import os
```

Comment:
@mjstealey Any idea why this newly added tests are not running on Jenkins? Is there any configuration that is needed for Jenkins to pick up added tests?